addons/DistancePlus/readme.md

```diff
@@ -0,0 +1,37 @@
+--[[
```

Comment:
The readme does not need a copyright notice, and it's unusual to have it (no other addon does). I'm not sure if that would even be valid, since the license is explicitly about code (`[…] in source and binary forms […]`).